### ISSUE TITLE
Fix `custom-property-no-missing-var-function` false positive for `timeline-scope`

### ISSUE THREAD

**Issue by splincode** [stylelint/stylelint#9163]
### What minimal example or steps are needed to reproduce the bug?

https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/timeline-scope

```css
@property --value {
    syntax: '<number>';
    inherits: true;
    initial-value: 0;
}

[directive] {
    timeline-scope: --value; // valid by https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/timeline-scope
}
```

### What minimal configuration is needed to reproduce the bug?

```js
module.exports = {
    $schema:
        'https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/stylelintrc.json',
    rules: {
          'custom-property-no-missing-var-function': true,
    },
};
```

### How did you run Stylelint?

```
stylelint projects/stylelint-config/tests/valid.css
```

### Which Stylelint-related dependencies are you using?

```
"peerDependencies": {
        "@stylistic/stylelint-config": "^4.0.0",
        "@stylistic/stylelint-plugin": "^5.0.1",
        "@taiga-ui/browserslist-config": "0.433.0",
        "postcss": "^8.5.8",
        "postcss-less": "^6.0.0",
        "stylelint": "^17.5.0",
        "stylelint-config-standard": "^40.0.0",
        "stylelint-no-unsupported-browser-features": "^8.1.1",
        "stylelint-order": "^8.1.1",
        "stylelint-plugin-logical-css": "^2.0.2",
        "stylelint-plugin-use-baseline": "^1.4.1",
        "stylelint-rem-over-px": "^1.0.2",
        "stylelint-use-logical": "^2.1.3"
    },
```

### What did you expect to happen?

No error

### What actually happened?

```bash
projects/stylelint-config/tests/valid.css
  20:21  ✖  Unexpected missing var function for "--value"  custom-property-no-missing-var-function
```

### Do you have a proposal to fix the bug?

No

**Comment by jeddy3** [comment]
@splincode Thanks for the report and for using the template.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/contributor-guide/rules#fix-a-bug-in-a-rule) in the Contributor guide.

Probably just need to add the property here:

https://github.com/stylelint/stylelint/blob/04d08d7c7adc4c96c5e3b634772365137538d5d6/lib/rules/custom-property-no-missing-var-function/index.mjs#L22-L42

**Comment by splincode** [comment]
@jeddy3 Hello, is it good?
https://github.com/stylelint/stylelint/pull/9164

**Comment by ybiquitous** [comment]
Fixed via #9164
